### ISSUE TITLE
File upload fails with incorrect error message

### ISSUE THREAD

**Issue by FelixKrueger** [DesmondWillowbrook/Librarian#24]
I tried to upload a full size gzip compressed FastQ file (one that worked as a subset in a previous test). Librarian produced this error message:
 <img width="1184" alt="Screenshot 2023-09-15 at 22 16 28" src="https://github.com/DesmondWillowbrook/Librarian/assets/5870509/fd30f460-63b6-4cb7-b487-088559bdbd36">

The file has 150bp long reads (untrimmed), is 23GB in size and contains 1,204,065,472 lines. It processes fine through FastQC, and is a fine file all-round.

**Comment by DesmondWillowbrook** [comment]
This might be a misconfiguration with the core FASTQ parser. You said that a subset of the file worked fine in a previous test; how does the subset differ from this file? Was the previously-tested subset the top _n_ lines or something else?

**Comment by FelixKrueger** [comment]
Indeed,  the subset was the first 1 million lines of the same file, also gzip compressed

**Comment by DesmondWillowbrook** [comment]
Note: I suspect some sort of overflow issue in the interop code between JS and the WebAssembly.
